src/cml.js

```diff
@@ -463,21 +463,26 @@ class CML {
   }
 
   async ci(opts = {}) {
-    const {
+    let {
+      fetchDepth = 1,
       unshallow = false,
       userEmail = GIT_USER_EMAIL,
       userName = GIT_USER_NAME,
       remote = GIT_REMOTE
     } = opts;
 
+    if (isNaN(fetchDepth) || fetchDepth < 0) {
```

Comment:
Since we set this as a number for yargs `isNaN` should not be needed. For `fetchDepth < 0` I think that we can handle this in a coerce function. There are a few examples in the repo you can look at.

Reply:
Done!